apps/challenges/admin.py

```diff
@@ -45,3 +51,10 @@ class LeaderboardDataAdmin(TimeStampedAdmin):
     list_display = ("challenge_phase_split", "submission", "leaderboard", "result")
     list_filter = ("challenge_phase_split", "submission", "leaderboard", "result")
     search_fields = ("challenge_phase_split", "submission", "leaderboard", "result")
+
+
+@admin.register(ChallengeConfiguration)
+class ChallengeConfigurationAdmin(TimeStampedAdmin):
+    list_display = ("user", "challenge", "zip_configuration", "is_created", "stdout_file", "stderr_file",)
+    list_filter = ("user", "challenge", "is_created",)
```

Comment:
Keeping a filter on challenge actually makes no sense because it is one to one field.